sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -198,6 +202,10 @@ export class AssessmentController {
         job.variables.assessmentId,
         job.variables.assessmentData,
       );
+      await this.disbursementScheduleService.updateDisbursementsHasEstimatedAwards(
```

Comment:
The `hasEstimatedAwards`  must be set in the disbursement controller where the disbursements are inserted into DB. It is a not nullable field.

[disbursement-schedule-shared.service.ts](https://github.com/bcgov/SIMS/blob/5a86bf249cad14b29d547a2e8ee3a831e0d677c6/sources/packages/backend/libs/services/src/disbursement-schedule/disbursement-schedule-shared.service.ts#L124)

Reply:
Also please add a E2E test for the worker. [disbursement.controller.saveDisbursementSchedules.e2e-spec](https://github.com/bcgov/SIMS/blob/main/sources/packages/backend/apps/workers/src/controllers/disbursement/_tests_/e2e/disbursement.controller.saveDisbursementSchedules.e2e-spec.ts)